envision/web/src/components/control_panel.js

```diff
@@ -30,8 +30,8 @@ export const attrs = Object.freeze({
 });
 
 export const agentModes = Object.freeze({
-  egoObs: "6",
```

Comment:
Why was this set as a string before?

Reply:
I am not sure, it evaluates different in javascript but not always.